.my-creds

```diff
@@ -0,0 +1 @@
+https://mohdabdin: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@github.com
```

Comment:
Not sure if this is something related to your credential, but if so I would HIGHLY recommend remove it to keep your account safe